blocks/library/cover-image/index.js

```diff
@@ -16,32 +11,18 @@ import classnames from 'classnames';
 import './editor.scss';
 import './style.scss';
 import { createBlock } from '../../api';
-import RichText from '../../rich-text';
-import AlignmentToolbar from '../../alignment-toolbar';
 import MediaUpload from '../../media-upload';
 import ImagePlaceholder from '../../image-placeholder';
 import BlockControls from '../../block-controls';
-import BlockAlignmentToolbar from '../../block-alignment-toolbar';
 import InspectorControls from '../../inspector-controls';
+import InnerBlocks from '../../inner-blocks';
 
 const validAlignments = [ 'left', 'center', 'right', 'wide', 'full' ];
 
 const blockAttributes = {
-	title: {
-		type: 'array',
-		source: 'children',
-		selector: 'p',
-	},
 	url: {
 		type: 'string',
 	},
-	align: {
```

Comment:
I think `align` should stay in here. Otherwise, it won't be possible to change the alignment of the container.

Reply:
Yes, the idea I have in mind is that it is no possible to change the alignment of the container. The container takes 100% width. So for example, if we want to right align text we apply the alignment on the nested paragraph block.